docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '2'
+
+services:
+  api:
+    build:
+      context: .
+      dockerfile: ./compose/api/Dockerfile
+    volumes:
+      - ./docs:/usr/src/app/docs
+      - ./tests:/usr/src/app/tests
+      - ./web:/usr/src/app/web
+      - ./setup.py:/usr/src/app/setup.py
+    env_file:
+      - .env
+    environment:
+      API_HOST: api
```

Comment:
Only took two plane rides and the layover inbetween to figure out I needed this...